Jenkinsfile.content

```diff
@@ -9,14 +26,20 @@ node('vetsgov-general-purpose') {
   dockerContainer = commonStages.setup()
 
   stage("Build") {
-    commonStages.build(params.ref, dockerContainer, params.ref, params.env, false)
+    commonStages.build(ref, dockerContainer, ref, params.env, false)
   }
 
   stage("Prearchive") {
     commonStages.prearchive(dockerContainer, params.env)
   }
 
   stage("Archive") {
-    commonStages.archive(dockerContainer, params.ref, params.env)
+    commonStages.archive(dockerContainer, ref, params.env)
+  }
+
+  stage("Deploy Dev or Staging") {
+    if (params.env != "vagovprod" && params.deploy) {
+      commonStages.runDeploy('deploys/vets-website-${params.env}', ref)
```

Comment:
Is it possible to send a slack message after this deploy is complete? I'm not sure how `runDeploy` works, but it'd be nice for them to know when the refresh was ready on staging/dev.

Reply:
yeah I think we can do that